cirro/config.py

```diff
@@ -81,8 +82,12 @@ def __init__(self, base_url: str = None):
         self.user_config = load_user_config()
         self.base_url = (base_url or
                          os.environ.get('CIRRO_BASE_URL') or
-                         (self.user_config.base_url if self.user_config else None) or
-                         Constants.default_base_url)
+                         (self.user_config.base_url if self.user_config else None))
+        if not self.base_url:
+            raise RuntimeError('No base URL provided, please provide the base_url parameter,'
+                               ' set the CIRRO_BASE_URL environment variable, or run cirro configure')
```

Comment:
How do you feel about switching the order of these options in the error msg to be cirro configure, then env variable, and then parameter?

Reply:
Yeah, will update that one also.